### PR TITLE
feat(nest): 👷 add ci and dockerfile

### PR DESCRIPTION
**Pull request by MiniMarker**

### CODE REVIEW

File: nest/.gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+      coverage_report:
+        coverage_format: cobertura
+        path: coverage/cobertura-coverage.xml
+    expire_in: 1 day
+  except:
+    - tags
+
+# SonarQube code analysis upload.
+# This overwrites the default script provided by included job: https://gitlab.intility.com/digital-telco/utils/gitlabci/-/blob/master/sonarqube/SonarQube-Template.gitlab-ci.yml
+# TODO: Insert link to SonarQube docs
```
Comment (dem92) [inline nest/.gitlab-ci.yml:66]: This should be replaced with a link to the docs

File: nest/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+RUN npm run build
+
+# ---- RUNTIME IMAGE ----
+
+FROM node:16-alpine as runtime
+
+## Set time zone config for Linux image
+ENV TZ="Europe/Oslo"
+
+ARG BUILD_VERSION=unknown
```
Comment (dem92) [inline nest/Dockerfile:22]: Can you add a code comment to explain why BUILD_VERSION is set to unknown? It is a bit unclear

**Review by dem92 — CHANGES_REQUESTED** [review]

File: nest/.gitlab-ci.yml (hunk 1 above)
Comment (MiniMarker) [reply nest/.gitlab-ci.yml:66]: But we dont have a doc yet :D

File: nest/Dockerfile (hunk 2 above)
Comment (MiniMarker) [reply nest/Dockerfile:23]: done :)

File: nest/.gitlab-ci.yml (hunk 1 above)
Comment (dem92) [reply nest/.gitlab-ci.yml:66]: Never mind then 😄

**Review by dem92 — APPROVED** [review]

**Pull request merged**